Animals/src/main/java/Main.java

```diff
@@ -21,22 +26,43 @@ public static void main(String[] args) {
 
         CreateAnimalService createAnimalService = new CreateAnimalService() {
             @Override
-            public void createAnimals() {
+            public Map<String, List<Animal>> createAnimals() {
                 CreateAnimalService.super.createAnimals();
+                return null;
```

Comment:
На всякий лучше возвращай new HashMap<>(), чтобы избавить от потенциальной npe

Reply:
Косяк, не досмотрел. Можно вообще вот так
```suggestion
                return CreateAnimalService.super.createAnimals();
```